### PR TITLE
Fix blood elf starting zones

### PR DESCRIPTION
**Pull request by Username5173**
**Changes proposed:**

-  Fixes creature scaling in eversong woods and ghostlands
-  Fixes 2 broken quests

**Issues addressed:** Closes #  (insert issue tracker number)

**Tests performed:** (Does it build, tested in-game, etc.)
-tested ingame

**Known issues and TODO list:** (add/remove lines as needed)

- [ ] 
- [ ]

### CODE REVIEW

File: sql/ashamane/world/2018_99_99_99_world_update.sql [hunk 1]
```diff
@@ -0,0 +1,158 @@
+REPLACE INTO `creature_template_scaling` (`Entry`, `LevelScalingMin`, `LevelScalingMax`, `LevelScalingDeltaMin`, `LevelScalingDeltaMax`, `VerifiedBuild`) VALUES (16294, 9, 20, 0, 2, NULL);
```
Comment (leviathonbeast) [inline sql/ashamane/world/2018_99_99_99_world_update.sql]: REPLACE INTO isnt good here

use 

UPDATE

File: sql/ashamane/world/2018_99_99_99_world_update.sql [hunk 2]
```diff
@@ -1,158 +1,161 @@
...
-REPLACE INTO `creature_template_scaling` (`Entry`, `LevelScalingMin`, `LevelScalingMax`, `LevelScalingDeltaMin`, `LevelScalingDeltaMax`, `VerifiedBuild`) VALUES (16308, 19, 61, 0, 2, NULL);
-REPLACE INTO `creature_template_scaling` (`Entry`, `LevelScalingMin`, `LevelScalingMax`, `LevelScalingDeltaMin`, `LevelScalingDeltaMax`, `VerifiedBuild`) VALUES (16300, 20, 60, 0, 2, NULL);
-REPLACE INTO `creature_template_scaling` (`Entry`, `LevelScalingMin`, `LevelScalingMax`, `LevelScalingDeltaMin`, `LevelScalingDeltaMax`, `VerifiedBuild`) VALUES (16349, 16, 60, 0, 2, NULL);
-REPLACE INTO `creature_template_scaling` (`Entry`, `LevelScalingMin`, `LevelScalingMax`, `LevelScalingDeltaMin`, `LevelScalingDeltaMax`, `VerifiedBuild`) VALUES (16292, 13, 60, 0, 2, NULL);
-REPLACE INTO `creature_template_scaling` (`Entry`, `LevelScalingMin`, `LevelScalingMax`, `LevelScalingDeltaMin`, `LevelScalingDeltaMax`, `VerifiedBuild`) VALUES (16162, 6, 20, 0, 2, NULL);
+INSERT INTO `creature_template_scaling` (`Entry`, `LevelScalingMin`, `LevelScalingMax`, `LevelScalingDeltaMin`, `LevelScalingDeltaMax`, `VerifiedBuild`) VALUES (16294, 9, 20, 0, 2, NULL);
```
Comment (Traesh) [inline sql/ashamane/world/2018_99_99_99_world_update.sql]: If you insert you have to delete before ^^

Comment (Username5173) [reply sql/ashamane/world/2018_99_99_99_world_update.sql]: There shouldn't be any entrys in the database i just used replace cuz i got them in mine.

Comment (Traesh) [reply sql/ashamane/world/2018_99_99_99_world_update.sql]: I know but a SQL patch should always be able to be applied multiple times

Comment (leviathonbeast) [reply sql/ashamane/world/2018_99_99_99_world_update.sql]: if there are no entries for those ids then still INSERT would be better then?

**Comment by leviathonbeast** [comment]
are you using sniffed data?

**Comment by Username5173** [comment]
I used wowhead but LevelScalingDeltaMax might be wrong.

**Comment by leviathonbeast** [comment]
ill go sniff the zone 
brb

**Comment by leviathonbeast** [comment]
@canewadar123 what quest is it that needs the orb of translocation?

**Pull request merged**